src/hotspot/share/gc/shenandoah/shenandoahLogFileOutput.hpp

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2013, 2021, Red Hat, Inc. All rights reserved.
```

Comment:
Is this a new file? Then the 2021-Amazon header should be enough. If it's derived from an existing file, then the header is ok as it is.